addOns/zest/src/main/java/org/zaproxy/zap/extension/zest/ZestScriptNode.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2025 The ZAP Development Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.zaproxy.zap.extension.zest;
+
+import org.zaproxy.zap.extension.script.ScriptNode;
+
+@SuppressWarnings("serial")
+public class ZestScriptNode extends ScriptNode {
+
+    public ZestScriptNode() {
+        super();
+    }
+
+    public ZestScriptNode(String name) {
+        super(name);
+    }
+
+    @Override
+    public Object getUserObject() {
+        return this.userObject;
+    }
```

Comment:
Actually, I just realized that this well not work as expected, because we can't change the return type to a broader type. Maybe I'll add dedicated setters / getters as part of this PR, changing it to draft until that is done.